src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,1177 @@
+        __m128i maskGB = _mm_setr_epi8(0x80, 1, 2, 0x80, 4, 5, 0x80, 7, 8, 0x80, 10, 11, 0x80, 13, 14, 0x80);
+        RpptROI roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
+
+        Rpp32u x_offset_r = batch_x_offset_r[batchCount];
+        Rpp32u y_offset_r = batch_y_offset_r[batchCount];
+        Rpp32u x_offset_g = batch_x_offset_g[batchCount];
+        Rpp32u y_offset_g = batch_y_offset_g[batchCount];
+        Rpp32u x_offset_b = batch_x_offset_b[batchCount];
+        Rpp32u y_offset_b = batch_y_offset_b[batchCount];
+
+        Rpp32u elementsInRowMax = srcDescPtr->w;
+
+        Rpp32u xOffsets[3] = {
+            x_offset_r,
+            x_offset_g,
+            x_offset_b};
+
+        Rpp32u yOffsets[3] = {
+            y_offset_r,
+            y_offset_g,
+            y_offset_b};
+
+        Rpp32u xOffsetsLoc[3] = {
+            x_offset_r,
+            x_offset_g,
+            x_offset_b};
+
+        Rpp32u yOffsetsLoc[3] = {
+            y_offset_r * elementsInRowMax,
+            y_offset_g * elementsInRowMax,
+            y_offset_b * elementsInRowMax};
+
+        Rpp8u *srcPtrImage, *dstPtrImage;
+        srcPtrImage = srcPtr + batchCount * srcDescPtr->strides.nStride;
+        dstPtrImage = dstPtr + batchCount * dstDescPtr->strides.nStride;
+
+        Rpp32u bufferLength = roi.xywhROI.roiWidth * layoutParams.bufferMultiplier;
+        Rpp8u *srcPtrChannel, *dstPtrChannel;
+        srcPtrChannel = srcPtrImage + (roi.xywhROI.xy.y * srcDescPtr->strides.hStride) + (roi.xywhROI.xy.x * layoutParams.bufferMultiplier);
+        dstPtrChannel = dstPtrImage;
+        Rpp32u alignedLength = (bufferLength / 32) * 32;
+        Rpp32u alignedLengthPerChannel = (srcDescPtr->w / 16) * 16;
+        Rpp32u vectorIncrement = 48;
+        Rpp32u vectorIncrementPerChannel = 16;
+        if ((srcDescPtr->c == 3) && (srcDescPtr->layout == RpptLayout::NHWC) && (dstDescPtr->layout == RpptLayout::NCHW ))
+        {
+            printf("%d ",roi.xywhROI.roiWidth);
+            convert_Pkd3_Pln3<Rpp8u>(srcPtrChannel, dstPtrChannel, srcDescPtr, dstDescPtr, roi.xywhROI.roiWidth * 3);
+            for(int c = 0; c < 3; c++)
+            {
+                Rpp32u offset = yOffsetsLoc[c] + xOffsetsLoc[c];
+                Rpp8u *srcPtrImageTemp, *dstPtrImageTemp;
+                srcPtrImageTemp = dstPtrImage + (c * dstDescPtr->strides.cStride) + yOffsetsLoc[c] + xOffsetsLoc[c];
+                dstPtrImageTemp = dstPtrImage + (c * dstDescPtr->strides.cStride);
+                if(offset > 0)
+                {
+                    for(int j = 0; j < roi.xywhROI.roiHeight; j++)
+                    {
+                        Rpp8u *srcTemp = srcPtrImageTemp;
+                        Rpp8u *dstTemp = dstPtrImageTemp;
+                        Rpp32u AligLen = ((alignedLengthPerChannel - xOffsetsLoc[c]) / 32) * 32;
```

Comment:
This computation can be moved outside line no 272 i.e outside the if(offset > 0) condition

Reply:
Do not use the variable name as AlignLen, please follow variable naming convention